portal-frontend/src/app/features/edit-submission/other-parcels/other-parcels.component.ts

```diff
@@ -254,7 +254,6 @@ export class OtherParcelsComponent extends StepComponent implements OnInit, OnDe
           } else {
             this.hasOtherParcelsInCommunity.patchValue('true');
             this.formDisabled = false;
-            await this.setHasOtherParcelsInCommunity(true);
```

Comment:
Can you explain why we don't want to do this?

Reply:
This was resetting and clearing the form when user wanted to cancel removing parcel.